src/lightning_app/cli/lightning_cli.py

```diff
@@ -370,6 +379,7 @@ def run():
     help="Decide whether to launch the app UI in a web browser",
 )
 @click.option("--env", type=str, default=[], multiple=True, help="Env variables to be set for the app.")
+@click.option("--secret", type=str, default=[], multiple=True, help="Secret variables to be set for the app.")
```

Comment:
probably? because for multiple variables we'll have to use the flag multiple times?
```suggestion
@click.option("--secret", type=str, default=[], multiple=True, help="Secret variable to be set for the app.")
```

Reply:
I was just following the same `help` argument being used for the `--env` option right above it.